src/parser.rs

```diff
@@ -501,8 +501,10 @@ mod tests {
             File {path: "tzu".into(), meta: None},
         ));
 
+        // note: the trailing tab after lao is intentional and significant here
+        // see https://github.com/uniphil/patch-rs/commit/364897dae8599cdb758c00fdc8dacd1645959070
         let sample2b = "\
---- lao
+--- lao	
```

Comment:
Can I suggest instead adding a `\t` into the source so that it's obvious to humans and so we don't risk losing it later?
```suggestion
--- lao\t
```